### PR TITLE
notifications_olgab

### PR DESCRIPTION
**Pull request by OlyaB**

### CODE REVIEW

File: topics/ui/controls/notification_types.md [hunk 1]
```diff
@@ -6,96 +6,45 @@
...
+    - Editor
+    - Tool window
+    - Dialog
+    - Any other location
+
+
+### Alert
+An action is required immediately, in any context.
+
+![](notification_type_alert.png){width=706}
```
Comment (eldar-jetbrains) [inline topics/ui/controls/notification_types.md:34]: An alt text is missing. I would suggest something like "Alert 'Open Project' asking where to open 'myJavaProject' with options to cancel, open in a new window, or this window"

File: topics/ui/controls/notification_types.md [hunk 2]
```diff
@@ -6,96 +6,45 @@
...
+
+### Banner
+[Main article](banner.md)
+
+An action is required but not immediately, in the editor, tool windows, and dialogs.
+![](notification_type_banner_action_required_editor.png){width=706}
```
Comment (eldar-jetbrains) [inline topics/ui/controls/notification_types.md:40]: I would also add an alt text here, something like "Banner in the editor with a warning 'Project JDK is not defined' and action 'Setup JDK'"

File: topics/ui/controls/notification_types.md [hunk 3]
```diff
@@ -6,96 +6,45 @@
...
+
+An action is not required in a dialog.
+![](notification_type_banner_action_required_dialog.png){width=706}
```
Comment (eldar-jetbrains) [inline topics/ui/controls/notification_types.md:43]: A possible alt text here could be "Banner in a dialog with a notification 'Interactive lesson available' and actions 'Open lesson' and an icon button to close the banner"

File: topics/ui/controls/notification_types.md [hunk 4]
```diff
@@ -6,96 +6,45 @@
...
+
+### Notification balloon
+[Main article](balloon.md)
+
+An action is not required to proceed, in any context except dialogs.
+
+![](notification_type_balloon_action_not_required.png){width=706}
```
Comment (eldar-jetbrains) [inline topics/ui/controls/notification_types.md:50]: Also a possible alt text here could be "A notification balloon warning about a shortcut conflict with a description and actions 'Modify shortcuts' and 'Don't show again'"

Comment (OlyaB) [reply topics/ui/controls/notification_types.md:50]: Thank you! Added alt texts everywhere

**Review by eldar-jetbrains — APPROVED** [review]
Checked the new version, everything is good from my side. Thank you!

File: topics/ui/controls/notification_types.md [hunk 5]
```diff
@@ -6,96 +6,43 @@
...
 - [Banner](banner.md)
-- [Balloon](balloon.md)
-- Tool window balloon
+- [Notification balloon](balloon.md)
 
 ## What notification to use
 
-Consider two factors when deciding which notification type to use.
-
-**User action**
```
Comment (MariannaKononenko) [inline topics/ui/controls/notification_types.md:20]: Let's add links to the corresponding subchapters

File: topics/ui/controls/notification_types.md [hunk 6]
```diff
@@ -6,96 +6,43 @@
...
+    - Dialog
+    - Any other location
+
+
+### Action is required immediately
+Use an alert in any context:
+![An alert 'Open Project' asking where to open 'myJavaProject' with options to cancel, open in a new window or this window](notification_type_alert.png){width=706}
+
+### Action is required but not immediately
+Use a [banner](banner.md) if the context is the editor, a tool window, or a dialog:
```
Comment (MariannaKononenko) [inline topics/ui/controls/notification_types.md:36]: here is not clear which banner to use in dialogs and toolwindows. If it is an ilnile banner, maybe we should write it explicitly and/or add more images

File: topics/ui/controls/notification_types.md [hunk 7]
```diff
@@ -6,96 +6,43 @@
...
+![A banner in the editor with a warning 'Project JDK is not defined' and action 'Setup JDK'](notification_type_banner_action_required_editor.png){width=706}
+
+A situation, where an action is required but not immediatly in any other context, is unlikely. However, if it occures, use a [notification balloon](balloon.md).
```
Comment (MariannaKononenko) [inline topics/ui/controls/notification_types.md:39]: Not sure if we should write about the probability of this event. IMO "In any other context, use a notification baloon." is enough.

File: topics/ui/controls/notification_types.md (hunk 6 above)
Comment (MariannaKononenko) [reply topics/ui/controls/notification_types.md:36]: Just saw a screenshot of a dialog below. If it is the same banner, then it's ok I guess

**Review by MariannaKononenko — APPROVED** [review]
LGTM, just few little comments

**Pull request merged**